soh/src/code/main.c

```diff
@@ -56,6 +56,16 @@ void Main(void* arg) {
     s32 debugHeapSize;
     s16* msg;
 
+#if defined(_WIN32)
+    if (CVar_GetS32("gCryptoRandom", 0)) {
+        if (!cryptoRandomInit()) {
+            osSyncPrintf("Unable to initialize CSPRNG. Quitting.");
```

Comment:
I would just fall back to the default RNG

Reply:
I've gone ahead and fixed that, as well as caught another issue that would have deviated from the base game RNG behavior. I think things look good now. Please let me know if there's anything else you'd like me to look at.